changelog/fragments/spec-scoreacrd-fix.yaml

```
@@ -0,0 +1,17 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (scorecard): Fixes a bug where an error would be thrown if a Custom Resource did not have a `.spec` field, which is a perfectly valid scenario.  Instead of throwing an error scorecard will now issue a warning.
```

Teeny nit - we should follow the language of scorecard:
```suggestion
      (scorecard): Fixes a bug where an error would be thrown if a Custom Resource did not have a `.spec` field, which is a perfectly valid scenario.  Instead of throwing an error scorecard will now issue a suggestion.
```